onnxruntime/python/tools/transformers/convert_generation.py

```diff
@@ -249,6 +249,15 @@ def parse_arguments(argv: Optional[List[str]] = None) -> argparse.Namespace:
     )
     model_group.set_defaults(past_present_share_buffer=False)
 
+    model_group.add_argument(
+        "--use_decoder_masked_multihead_attention",
+        required=False,
+        action="store_true",
+        help="Uses `DecoderMaskedMultiheadAttention` to optimize the unidirectional decoding Attention computation. "
+        "Must be used with `past_present_share_buffer`",
+    )
+    model_group.set_defaults(use_decoder_masked_multihead_attention=False)
```

Comment:
Turned off by default just like `past_present_share_buffer`. Will make it default in future after we are confident about the accuracy.